### DISCUSSION TITLE
Wrong line number declaration on Java fields

### DISCUSSION THREAD

**Discussion by flozender** [community/community#57353] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Selecting the field [`moduleClassLoader`](https://github.com/checkstyle/checkstyle/blob/119fd4fb33bef9f5c66fc950396669af842c21a3/src/main/java/com/puppycrawl/tools/checkstyle/Checker.java#L102) pops open the definitions tab on the right side, however, the line number shown there is wrong (102), where as the field was actually declared on line 108. In the screenshot, you can also see the highlighting it did on click, which is wrong. 

![image](https://github.com/community/community/assets/40818234/3cbdf8a0-a5eb-4ffe-8b54-3eefdf19a241)

Also, selecting a field with less than three characters doesn't even open up the definitions tab. Try this [example](https://github.com/javaparser/javaparser/blob/97555053af3025556efe1a168fd7943dac28a2a6/javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/SourceFileInfoExtractor.java#L58) to reproduce this bug.

**Comment by look** [comment]
Thanks for the report. I replicated it and forwarded it to the team responsible for this.

**Reply by flozender** [reply]
Hi Luke, do you happen to have any updates on when a fix could potentially be deployed for this issue? Thanks!

**Reply by look** [reply]
Hi @flozender, sorry for the delay. We're aware of the problem and are working on a fix. The problem is that you are getting results for the default branch, but that's not the version of the file you're looking at.
